pymongo/lock.py

```diff
@@ -28,6 +33,14 @@
 
 _T = TypeVar("_T")
 
+# Needed to support 3.13 asyncio fixes in older versions of Python
```

Comment:
Can you add `https://github.com/python/cpython/issues/112202` here? That's the most important bugfix we need.